src/NSwag.MSBuild.CodeGeneration/NSwag.MSBuild.CodeGeneration.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>NSwag.MSBuild.CodeGeneration</id>
+    <version>11.20.1</version>
+    <authors>Rico Suter</authors>
+    <owners>Rico Suter</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/NSwag/NSwag/blob/master/LICENSE.md</licenseUrl>
+    <projectUrl>https://github.com/NSwag/NSwag</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/NSwag/NSwag/master/assets/NuGetIcon.png</iconUrl>
+    <description>NSwag: The Swagger API toolchain for .NET and TypeScript</description>
+    <tags>Swagger Documentation WebApi AspNet TypeScript CodeGen</tags>
+    <developmentDependency>true</developmentDependency>
+    <dependencies>
+      <dependency id="Microsoft.Extensions.ApiDescription.Design" Version="2.2.0-preview3-t007163513" />
```

Comment:
Will a nuget.org push work or is it validation for that?

Reply:
Nothing breaks when building this repo. But, users won't be able to consume the new package until the versions line up on NuGet.org. 

I'll update this to use a version range. Unfortunately, a version range here mainly helps initially because NuGet will pick the _lowest_ version it finds that's in the range. Users would need to reference `Microsoft.Extensions.ApiDescription.Design` directly in their `.csproj` file to get something newer e.g. a nightly build from https://dotnet.myget.org/gallery/aspnetcore-dev.

Reply:
> I'll update this to use a version range. 

Will you update this in this PR or in a new PR?

Reply:
Unfortunately, version ranges don't work with preview releases due to a NuGet bug. But, I'll update this PR to use a version available on http://dotnet.myget.org